docs/prd/PRD-0002-composition-lifecycle.md

```
@@ -0,0 +1,135 @@
+3. **List version history** — Governance workflows need to enumerate all versions of a composition to display audit trails, compare changes, or roll back.
+
+---
+
+## Requirements
+
+### Functional Requirements
+
+#### FR-1: Update Composition
+
+| Field | Detail |
+|---|---|
```

markdownlint-cli2 (0.20.0)

[warning] Table column style
Table pipe is missing space to the right for style "compact"

(MD060, table-column-style)